packages/cosmwasm/src/index.ts

```diff
@@ -1,6 +1,3 @@
-import * as logs from "./logs";
-export { logs };
-
```

Comment:
This is th point where we break a public API. Not sure anyone would notice, but still